### ISSUE TITLE
Does not detect broken embedded links

### ISSUE THREAD

**Issue by claremacrae** [ipshing/obsidian-broken-links#7]
Hi, I found that this is not recognised as a broken link:

```
![[Catch2 and CMake#Catch2 and CMake]]
```

When viewed in Reading mode it looks like this:

![image](https://github.com/ipshing/obsidian-broken-links/assets/4840096/c680578b-c88f-4883-9d93-b4096e7d1b3e)

Whilst this IS recognised as broken (the file does not exist):

```
[[Catch2 and CMake#Catch2 and CMake]]
```

So there seems to be something different in how Obsidian handles embedded/transcluded files... I've no idea if it is in scope for this project, but thought you might like to know...

**Comment by claremacrae** [comment]
Image files can also be embedded, and be broken links that Obsidian only shows as Helen when the file is rendered.

I don’t know if detecting this would be a separate request from this.

**Comment by ipshing** [comment]
> I don’t know if detecting this would be a separate request from this.

I think all embedded links are grouped together so there's no need to make it a separate bug.

**Comment by claremacrae** [comment]
Oh wow, BRAT just updated my install with the 1.2.0 release, and your fix of this found broken embeds of markdown sections, a few PNGs and even an excalidraw file.

Many thanks!

**Comment by ipshing** [comment]
This is fixed in version [1.2.0](https://github.com/ipshing/obsidian-broken-links/releases/tag/1.2.0). It should find any broken embedded links, regardless of file type (as long as Obsidian considers it an embedded link, that is).
